backend/src/main/resources/reference.conf

```diff
@@ -1,5 +1,5 @@
 FRONTEND_URL = "http://localhost:3000/SoftwareEngineering-WS2025-just-donate"
 
 MAIL_SMTP_HOST = "smtp.gmail.com"
-MAIL_ADDRESS = ""
-MAIL_PASSWORD = ""
+# MAIL_ADDRESS = ""
```

Comment:
The program wont start with these commented out. Either put back in or handle in config properly.